src/types.ts

```diff
@@ -25,3 +25,25 @@ export enum SelectionType {
   Hide,
   Highlight,
 }
+
+export enum AgentType {
+  Agent,
+  DisplayState,
```

Comment:
this is a little confusing because the viewer also has `agentType` which means something really different. I think something like a util that tells `hasChildren` or `isChild` would be more clear

Reply:
Does it mean something really different? I thought agent type and tags were basically the way that agent/display state were described when decoded from sim file format.

I’m definitely open to using the hasChildren isChild language